ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -191,6 +199,7 @@ protected ClientArgs(String args[]) {
       this.sslPropsFilePath = options.valueOf(sslPropsFilePath);
       this.targetAccountName = options.valueOf(targetAccountName);
       this.targetContainerName = options.valueOf(targetContainerName);
+      this.customizedHeader = options.valueOf(customizedHeader);
```

Comment:
joptsimple has a method that allows you to get multiple values of the same option (i.e.` --customHeader a:b --customHeader c:d`): `options.valuesOf(OptionSpec)`. You could use this to support multiple custom headers.

Reply:
+1